docs/README.md

```diff
@@ -121,6 +122,10 @@ Use the following URL in all HTTP requests that you send to the resource aggrega
 - {odimra_host} is the fully qualified domain name (FQDN) used for generating certificates while deploying the
     resource aggregator.
 	
+	To access the API endpoint using a REST client, do any one of the following:
+    - Use IP address of the system where the resource aggregator is installed as {odim_host}.
+    - To use FQDN as {odim_host}, add the Resource Aggregator for ODIM server certificate to the browser where REST client is launched.
```

Comment:
Not clear with the above two statements mentioned for accessing the API endpoint . 
Can we rephrase the statement?

Reply:
I will make changes in the pdf version first. Since these comments are not specific to open source content, we can mark these as resolved for now.